RELEASING.md

```
@@ -51,6 +52,47 @@ manually deployed using `mvn clean deploy`
     * **Always `clean` first.** The javadoc goals skip when their output is already present, so a
       re-run without `clean` can report `BUILD SUCCESS` without checking anything.
 
+### Pre-release bug hunt
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)